appinventor/components/src/com/google/appinventor/components/runtime/BluetoothConnectionBase.java

```diff
@@ -53,13 +53,14 @@
   private Object connectedBluetoothSocket;
   private InputStream inputStream;
   private OutputStream outputStream;
-
+  private int sdkLevel;
```

Comment:
You could make this final since it will always be the same for a given instance of the app.